core/framework/embedders.py

```
@@ -0,0 +1,997 @@
+                self.model_name,
+                trust_remote_code=True,
+                torch_dtype=dtype,
+                device_map=self.device
+            )
+        else:
+            self.model = AutoModel.from_pretrained(
+                self.model_name,
+                trust_remote_code=True,
+                torch_dtype=dtype
+            )
+        
```

Ruff (0.12.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)